src/main/resources/db/migration/V8__Add_participation_message_id_column_to_Polls_table.sql

```diff
@@ -0,0 +1 @@
+alter table polls add column participation_id varchar(36);
```

Comment:
The name is unclear, it's not easy to understand that this refers to an actual messageId from the Wire backend.
Maybe participation_message_id